### PR TITLE
docs(seed-docs): alert dialog

### PR DESCRIPTION
**Pull request by junghyeonsu**

### CODE REVIEW

**Comment by junghyeonsu** [comment]
1. What Changed
    - Added new images for overflow and minimum width styles of the alert dialog component.
    - Updated the status of the `react` and `docs` sections in `component-meta.json` from `todo` to `in-progress` and `done` respectively.
    - Added a new section for the design decisions related to color, typography, and layout in `style.mdx`.
    - Added a new section for usage options and interaction guidelines in `usage.mdx`.
    - Updated the description field in `primitive-meta.json`.
    - Added a new section for anatomy, states, contexts, actions, and triggers in `primitive.mdx`.

2. Impact
    - Developers can refer to the added images in `style.mdx` while implementing the styles for the alert dialog component.
    - The updated status of `component-meta.json` gives a better idea of the work progress for the component.
    - Developers can use the design decisions provided in `style.mdx` as guidelines for consistent visual design across the application.
    - The added usage options and interaction guidelines in `usage.mdx` provide better clarity on how to use the component.
    - The updated description in `primitive-meta.json` adds more information about the component for developers.
    - The added anatomy, states, contexts, actions, and triggers sections in `primitive.mdx` help developers understand the functionality of the dialog primitive and implement it easily.

3. Testing
    - Verify that the new images in `style.mdx` reflect the correct styles of the alert dialog component.
    - Verify that the updated status of `component-meta.json` is correct and gives an accurate description of the work progress for the component.
    - Verify that the design decisions provided in `style.mdx` are consistent with the application's visual design and are easy to implement.
    - Verify that the usage options and interaction guidelines in `usage.mdx` are clear and easy to follow.
    - Verify that the added anatomy, states, contexts, actions, and triggers sections in `primitive.mdx` accurately describe the functionality of the dialog primitive and help in its implementation.

**Review by malangcat — APPROVED** [review]

File: docs/content/component/alert-dialog/style.mdx [hunk 1]
```diff
@@ -2,4 +2,158 @@
...
+9. Secondary Action - Button Primitive
+10. Secondary Action Label - Button Primitive
+
+## Visual Options
+
+| Property                | Value                              | Description                     |
+| ----------------------- | ---------------------------------- | ------------------------------- |
+| title                   | text                               | 타이틀에 대한 옵션              |
+| description             | text                               | 설명에 대한 옵션                |
+| secondary action intent | alternative, neutral, nonpreffered | Secondary 버튼 위계에 대한 옵션 |
```
Comment (malangcat) [inline docs/content/component/alert-dialog/style.mdx:24]: issue: 위계 -> 의도
suggestion: Description column을 아예 제거해도 괜찮아요.

File: docs/content/component/alert-dialog/style.mdx [hunk 2]
```diff
@@ -2,4 +2,158 @@
...
+
+| Part                              | Attribute          | Value |
+| --------------------------------- | ------------------ | ----- |
+| Content                           | Min Width          | 272pt |
+|                                   | Vertical Padding   | 20pt  |
+|                                   | Horizontal Padding | 16pt  |
+|                                   | Corder Radius      | 16pt  |
+| Text Container / Action Container | Gap                | 16pt  |
+| Title / Description               | Gap                | 6pt   |
+| Primary Action / Secondary Action | Gap                | 8pt   |
```
Comment (malangcat) [inline docs/content/component/alert-dialog/style.mdx:106]: issue: Gap -> Spacing
```suggestion
| Text Container / Action Container | Spacing                | 16pt  |
| Title / Description               | Spacing                | 6pt   |
| Primary Action / Secondary Action | Spacing                | 8pt   |
```

File: docs/content/component/alert-dialog/style.mdx [hunk 3]
```diff
@@ -2,4 +2,158 @@
...
+| Part             | Attribute       | Value |
+| ---------------- | --------------- | ----- |
+| Action Container | Align Direction | Row   |
+
+### secondary action intent=alternative
+
+| Part             | Attribute       | Value |
+| ---------------- | --------------- | ----- |
+| Action Container | Align Direction | Row   |
+
```
Comment (malangcat) [inline docs/content/component/alert-dialog/style.mdx:144]: issue: Layout 내용 중간에 Shadow가 들어갔어요.
```suggestion
### secondary action intent=nonpreffered

| Part             | Attribute       | Value  |
| ---------------- | --------------- | ------ |
| Action Container | Align Direction | Column |

### secondary action intent=neutral

| Part             | Attribute       | Value |
| ---------------- | --------------- | ----- |
| Action Container | Align Direction | Row   |

### secondary action intent=alternative

| Part             | Attribute       | Value |
| ---------------- | --------------- | ----- |
| Action Container | Align Direction | Row   |

## 디자인 결정 (Shadow)

| Part    | Attribute   | Value               |
| ------- | ----------- | ------------------- |
| Content | Offset      | 0pt, 5pt            |
|         | Blur Radius | 15pt                |
|         | Spread      | 0pt                 |
|         | Color       | rgba(0, 0, 0, 0.1)  |
| Content | Offset      | 0pt, 9pt            |
|         | Blur Radius | 46pt                |
|         | Spread      | 0pt                 |
|         | Color       | rgba(0, 0, 0, 0.12) |
| Content | Offset      | 0pt, 10pt           |
|         | Blur Radius | 38pt                |
|         | Spread      | 0pt                 |
|         | Color       | rgba(0, 0, 0, 0.15) |
```

File: docs/content/primitive/dialog/primitive.mdx [hunk 4]
```diff
@@ -2,14 +2,41 @@
 slug: /primitive/dialog
 ---
 
-## Anatomy
+## 아나토미
```
Comment (malangcat) [inline docs/content/primitive/dialog/primitive.mdx:5]: issue: 네이밍 유지해주세요.

Comment (malangcat) [reply docs/content/primitive/dialog/primitive.mdx:5]:
```suggestion
## Anatomy
```

**Review by malangcat — CHANGES_REQUESTED** [review]

**Pull request merged**